src/building/how-to-build-and-run.md

```diff
@@ -52,14 +57,75 @@ If you have already built `rustc` and you change settings related to LLVM, then
 execute `rm -rf build` for subsequent configuration changes to take effect. Note that `./x.py
 clean` will not cause a rebuild of LLVM.
 
+Finally, rustbuild makes use of the [cc-rs crate] which has [its own
+method][env-vars] of configuring C compilers and C flags via environment
+variables.
+
+[cc-rs crate]: https://github.com/alexcrichton/cc-rs
+[env-vars]: https://github.com/alexcrichton/cc-rs#external-configuration-via-environment-variables
```

Comment:
this is the sort of thing I mean by "not user-facing" - if you want to add this information I would probably put it in https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html#passing-flags-to-commands-invoked-by-bootstrap, not in the intro.

Reply:
Reverted it along with the information above